frontend/public/locales/fr/data-unavailable.json

```
@@ -6,9 +6,8 @@
   "service-eligible": "<strong>Vous n'êtes pas un participant du Régime canadien de soins dentaires actuellement.</strong> Pour savoir si vous êtes admissible, consultez le site <doyouqualify>Êtes-vous admissible</doyouqualify>, sur Canada.ca. Pour obtenir de plus amples renseignements sur le processus de demande, veuillez visiter <howtoapply>Présentation d'une demande</howtoapply>.",
   "do-you-qualify.href": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/admissibilite.html",
   "how-to-apply.href": "https://www.canada.ca/fr/services/prestations/dentaire/regime-soins-dentaires/demande.html",
-  "other-enquiry": "Pour d'autres demandes de renseignements, visitez la page <contactus>Contactez-nous</contactus> pour savoir comment nous joindre.",
+  "other-enquiry": "Pour d’autres demandes concernant le Régime canadien de soins dentaires, veuillez visiter la page <contactus>Contactez-nous du RCSD</contactus>.",
```

Use normal `'` appostrophe instead of `’`.

```suggestion
  "other-enquiry": "Pour d'autres demandes concernant le Régime canadien de soins dentaires, veuillez visiter la page <contactus>Contactez-nous du RCSD</contactus>.",
```

---

[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/compare/e837391d482cfcc11fba6b5e02a217e9cfc1ccc2..96a5304badc0ea07f60db8d8e35eb335b81552b6)